doozer/doozerlib/cli/images_streams.py

```diff
@@ -714,8 +714,8 @@ def reconcile_jira_issues(runtime, pr_map: Dict[str, PullRequest.PullRequest], d
     for distgit_key, pr in pr_map.items():
         image_meta: ImageMetadata = runtime.image_map[distgit_key]
         potential_project, potential_component = image_meta.get_jira_info()
-        summary = f"Update {release_version} {image_meta.get_component_name()} image to be consistent with ART"
-        old_summary_format = f"Update {release_version} {image_meta.name} image to be consistent with ART"
+        summary = f"ART requests updates to {release_version} image {image_meta.get_component_name()}"
```

Comment:
I think we might be relying on this wording in some doozer commands (images:streams), so changing this could break them